### ISSUE TITLE
Attribute base_url in installed-plugins

### ISSUE THREAD

**Issue by TitouanS31** [nautobot/nautobot#6027]
### As ...

Nelly - Network Engineer

### I want ...

To add a `base_url` attribute to the plugins object in `plugins/installed-plugins`.

### So that ...

I have an explicit link between the plugin informations and its API entry point.

### I know this is done when...

I can programmatically fetch installed plugins, retrieve informations about them such as version and list their endpoints. For example, I would like an explicit way to know **BGP Models** entry point is `bgp`.

### Optional - Feature groups this request pertains to.

- [X] Automation
- [ ] Circuits
- [ ] DCIM
- [ ] IPAM
- [ ] Misc (including Data Sources)
- [ ] Organization
- [X] Apps (and other Extensibility)
- [ ] Security (Secrets, etc)
- [ ] Image Management
- [ ] UI/UX
- [ ] Documentation
- [ ] Other (not directly a platform feature)

**Comment by glennmatthews** [comment]
Note that `/api/plugins/` (for example https://demo.nautobot.com/api/plugins/) does list the registered app endpoints already, but it doesn't directly link against `/api/plugins/installed-plugins/`.

**Comment by glennmatthews** [comment]
Not directly related, but I noticed that `/api/plugins/installed-plugins/` (and its mirror `/api/apps/installed-apps/`) are still requiring `IsAdminUser` permissions, which is unnecessarily restrictive especially after #5353.

**Comment by TitouanS31** [comment]
> Note that `/api/plugins/` (for example https://demo.nautobot.com/api/plugins/) does list the registered app endpoints already, but it doesn't directly link against `/api/plugins/installed-plugins/`.

Yes, that is precisely the link I would like. For now there is no relation between a plugin and its endpoints.
